go.mod

```diff
@@ -62,10 +62,14 @@ require (
 )
 
 replace (
-	github.com/containers/storage => github.com/containers/storage v1.13.3-0.20191209173733-4822c6053840
+	github.com/containerd/containerd => github.com/containerd/containerd v0.2.10-0.20170808145631-06b9cb351610
```

Comment:
@nalind I had issues when I using newer versions of containerd. Have you found the same?

Reply:
I did, yes.

Reply:
@nalind how did you choose this specific version? It seems this refers to a SHA that is not associated with a tag, nor is it the latest commit in the `v0.2.x` branch.

Reply:
It was the version referenced by the `vendor.conf` in the version of `github.com/docker/docker` tree that we're using.  But you're right that there are later commits in the `v0.2.x` branch, and moving to its tip (`408d13de2fbb2298a9ae7c14d86803a740ab12a9`) doesn't appear to break compilation.  Should I update to point to that instead?

Reply:
@nalind let's do that, and add a comment that we're using the tip of that branch (ex: `# Result of go get github.com/containerd/containerd@v0.2.x`)

Reply:
Okay, going with that.